README.Rmd

```diff
@@ -89,11 +89,11 @@ pl %>% learn()
 
 ### Cross-validation pairs
 
-Cross-validation pairs can be customized with `learn_cvpairs()`. The following implements k-fold cross-validation, creating five folds:
+Cross-validation pairs can be customized with `learn_cvpairs()`. The following implements cross-validation with 5 random samples of 10 elements in each training set.
 
 ```{r}
 pl %>%
-  learn_cvpairs(k = 5) %>% 
+  learn_cvpairs(resamplr::holdout_n, size = 10, K = 5) %>% 
```

Comment:
Please stick to the modelr functions in the README which, after importing (see other comment) can be referenced without `modelr::`. You're welcome to create a vignette to expand on this functionality, but README is already a bit too long to bring in novel aspects like this.